scripts/us_census/acs5yr/subject_tables/common/generate_col_map.py

```diff
@@ -84,12 +84,18 @@ def process_zip_file(zip_file_path,
         "<column-name-2>": {}, .....,
       }
   """
+    zip_file_path = os.path.expanduser(zip_file_path)
+    spec_path = os.path.expanduser(spec_path)
+    output_dir_path = os.path.expanduser(output_dir_path)
```

Comment:
Not sure if we need to expand paths relative to user's home directory

Reply:
kept it only in process.py